adafruit_st7735.py

```diff
@@ -66,16 +96,75 @@
     # 1 clk cycle nonoverlap, 2 cycle gate rise, 3 sycle osc equalie,
     # fix on VTL
     b"\x3a\x01\x05" # COLMOD - 16bit color
-    b"\xe0\x10\x02\x1c\x07\x12\x37\x32\x29\x2d\x29\x25\x2B\x39\x00\x01\x03\x10" # _GMCTRP1 Gamma
-    b"\xe1\x10\x03\x1d\x07\x06\x2E\x2C\x29\x2D\x2E\x2E\x37\x3F\x00\x00\x02\x10" # _GMCTRN1
+)
+
+_INIT_R2_GREEN = (
     b"\x2a\x03\x02\x00\x81" # _CASET XSTART = 2, XEND = 129
     b"\x2b\x03\x02\x00\x81" # _RASET XSTART = 2, XEND = 129
+)
+
+_INIT_R2_RED = (
+    b"\x2a\x00\x00\x00\x7F" # _CASET XSTART = 0, XEND = 127
+    b"\x2b\x00\x00\x00\x9F" # _RASET XSTART = 0, XEND = 159
+)
+
+_INIT_R2_GREEN_144 = (
+    b"\x2a\x00\x00\x00\x7F" # _CASET XSTART = 0, XEND = 127
+    b"\x2b\x00\x00\x00\x7F" # _RASET XSTART = 0, XEND = 127
+)
+
+_INIT_R2_GREEN_160X80 = (
+    b"\x2a\x00\x00\x00\x4F" # _CASET XSTART = 0, XEND = 79
+    b"\x2b\x00\x00\x00\x9F" # _RASET XSTART = 0, XEND = 159
+)
+
+_INIT_R3 = (
+    b"\xe0\x10\x02\x1c\x07\x12\x37\x32\x29\x2d\x29\x25\x2B\x39\x00\x01\x03\x10" # _GMCTRP1 Gamma
+    b"\xe1\x10\x03\x1d\x07\x06\x2E\x2C\x29\x2D\x2E\x2E\x37\x3F\x00\x00\x02\x10" # _GMCTRN1
     b"\x13\x80\x0a" # _NORON
     b"\x29\x80\x64" # _DISPON
 )
 
+_INIT_R4_GREEN_160X80 = (
+    b"\x36\x01\x60" # _MADCTL Rotate to Landscape Mode
+)
+
+_INIT_R4_BLACK = (
+    b"\x36\x01\xC0" # _MADCTL Rotate to Landscape Mode
+)
```

Comment:
Having all of these bytearrays in one file will take more memory that strictly necessary. I'd rather have multiple files with some duplication.

Reply:
Ok, that makes sense.